### ISSUE TITLE
Did you guys forgot that you released 0.13.0 already?

### ISSUE THREAD

**Issue by thekalinga** [razorpay/razorpay-android-sample-app#15]
Hello,

There is a new release by you with the same version 0.13.0 as what you have released earlier.

Now
--
https://github.com/razorpay/razorpay-android-sample-app/releases/tag/v0.13.0

26th May
--
https://github.com/razorpay/razorpay-android-sample-app/commit/71e22cd97e907dfeddfc1c60d3aee34ccbe841d9

**Comment by shashankmehta** [comment]
@thekalinga Nope, we didn't forget. We did change the history of the `master` branch though. But the `v0.13.0` tag is still on the exact same commit and the release hasn't been changed. For some reason GitHub updated the date in the releases ¯\_(ツ)_/¯.

In any case, we are releasing `v1.0.0` which will be the latest version. I have also updated the release log to ensure there's no confusion. Thanks for informing!

tl;dr: `v0.13.0` is the same version that was released on 26th May.
